File: Aula 03/Java/calculadora.java
import java.util.Locale;
import java.util.Scanner; // Importa biblioteca para leitura de dados

public class calculadora {
    public static void main(String[] args) {
        // Configura o ponto como separador decimal (Padrão americano/programação)
        Locale.setDefault(Locale.US);

        // Passo 1: Instanciar o objeto Scanner para der a entrada do teclado
        Scanner sc = new Scanner(System.in);

        // Passo 2: Entrada de dados e declaração das variáveis

        System.out.println("--- Calculadora de Negócios (Java) ---");

        System.out.println("Digite o primeiro número: ");
        double numero1 = sc.nextDouble();

        System.out.println("Digite o segundo número: ");
        double numero2 = sc.nextDouble();

        // Passo 3: Processamento (Cálculos)
        double soma = numero1 + numero2;
        double subtracao = numero1 - numero2;
        double multiplicacao = numero1 * numero2;
        double divisao = numero1 / numero2;

        // Passo 4: Saída de resultados formatados
        System.out.printf("Soma: %.2f%n", soma);
        System.out.printf("Subtração: %.2f%n", subtracao);
        System.out.printf("Multiplicação: %.2f%n", multiplicacao);
        System.out.printf("Divisão: %.2f%n", divisao);
        
        // Passo 5: Fechar o Scanner (Boa prática de gerenciamento de memória)
        sc.close();
    }
}

File: Aula 03/C++/calculadora.cpp
#include <iostream> // Biblioteca para entrada e saída de dados
#include <iomanip> // Biblioteca para formatar a saída de casas decimais
using namespace std;

int main(){
    // Passo 1: Declaração de variáveis (Tipagem Estática)
    int numero1 = 0, numero2 = 0, soma, subtracao, multiplicacao;
    float divisao;

    // Passo 2: Entrada de dados
    cout << "Digite o primeiro número: "; cin >> numero1;
    cout << "Digite o segundo número: "; cin >> numero2;

    // Passo 3: Processamento (Cálculos Aritméticos)
    soma = numero1 + numero2;
    subtracao = numero1 - numero2;
    multiplicacao = numero1 * numero2;
    divisao = numero1 / numero2;

    // Passo 4: Saída de resultados formatados
    cout << fixed << setprecision(2);
    cout << "Soma: " << soma << endl;
    cout << "Subtração: " << subtracao << endl;
    cout << "Multiplicação: " << multiplicacao << endl;
    cout << "Divisão: " << divisao << endl;

    return 0; // Indica que o programa terminou com sucesso
}
